packages/api/src/auth/decoders/azureActiveDirectory.ts

```diff
@@ -19,7 +19,11 @@ const verifyAzureActiveDirectoryToken = (
       bearerToken,
       (header, callback) => {
         client.getSigningKey(header.kid as string, (error, key) => {
-          callback(error, key.getPublicKey())
+          try {
+            callback(error, key.getPublicKey());
+          } catch (err) {
+            console.error('An error occured while trying to obtain signing key from Azure Active Directory. This might be a result of a outage. See https://status.azure.com/en-us/status for current status.', err)
```

Comment:
Is the linter OK with this line? Looks really long

Reply:
It does not seem to do any changes when I manually do `Format Document` (and I have "Formatting on Save"). I agree, it should do something about that. Broken on my side?

Reply:
I don't understand why GH only runs two actions on this PR. I'm missing the linter action. Can you try changing something/anything and do another push to the PR branch?

Reply:
Now we have linting warnings! :)

Reply:
By any chance you can help me figure out why it's not linting locally for me? If so, hit me up on Discord DM :)